OracleSOASuite/dockerfiles/buildDockerImage.sh

```diff
@@ -110,6 +114,10 @@ then
   THEDIR=${VERSION}
 fi
 
+if [ "${PATCHING}" = "true" ]; then
+  DOCKERFILE_NAME=Dockerfile.patch
```

Comment:
This means patching only works if the customer uses `buildDockerImage.sh`. You should consider splitting the Dockerfile out to a different location (like the WebLogic examples do) so that it's more easily consumed by customers with existing CI/CD tooling who don't use our build scripts.

Basically, you want to ensure that running `docker build` works just as well as running the script we provide for convenience purposes only.

Reply:
This "Dockerfile.patch" file can also be used as a standalone script to build the images.

$ docker build  -t oracle/soa:12.2.1.3 -f Dockerfile.patch .

Reply:
Please add this to your documentation.